### PR TITLE
Feature/query helper

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
### Description of the Change
Introduce IrohaQueryHelper for easy mocking of Iroha requests.

### CODE REVIEW

File: notary-commons/src/main/kotlin/com/d3/commons/sidechain/iroha/util/impl/IrohaQueryHelperImpl.kt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import jp.co.soramitsu.iroha.java.IrohaAPI
+import jp.co.soramitsu.iroha.java.Query
+import jp.co.soramitsu.iroha.java.QueryAPI
+import java.security.KeyPair
+import java.util.concurrent.atomic.AtomicLong
+
+/**
+ * The purpose of the class is to hide Iroha query implementation.
+ */
+class IrohaQueryHelperImpl(val queryAPI: QueryAPI) : IrohaQueryHelper {
```
Comment (Mingela) [inline notary-commons/src/main/kotlin/com/d3/commons/sidechain/iroha/util/impl/IrohaQueryHelperImpl.kt]:
```suggestion
class IrohaQueryHelperImpl(private val queryAPI: QueryAPI) : IrohaQueryHelper {
```

File: btc/src/main/kotlin/com/d3/btc/provider/generation/BtcPublicKeyProvider.kt [hunk 2]
```diff
@@ -19,7 +18,7 @@ import org.springframework.stereotype.Component
 
 /**
  *  Bitcoin keys provider
- *  @param walletFile - bitcoin wallet
+ *  @param keysWallet - bitcoin wallet
```
Comment (dolgopolovwork) [inline btc/src/main/kotlin/com/d3/btc/provider/generation/BtcPublicKeyProvider.kt]: This is strange

File: notary-iroha-integration-test/src/integration-test/kotlin/integration/iroha/IrohaQueryHelperTest.kt [hunk 3]
```diff
@@ -5,15 +5,14 @@ import org.junit.jupiter.api.AfterAll
...
-import com.d3.commons.sidechain.iroha.util.getAccountAsset
 
 /**
  * Test helper class for Iroha queries
  *
  * Note: Requires Iroha running.
  */
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-class RequestHelperTest {
+class IrohaQueryHelperTest {
```
Comment (dolgopolovwork) [inline notary-iroha-integration-test/src/integration-test/kotlin/integration/iroha/IrohaQueryHelperTest.kt]: Could you please add more tests?

Comment (Alexey-N-Chernyshov) [reply notary-iroha-integration-test/src/integration-test/kotlin/integration/iroha/IrohaQueryHelperTest.kt]: Added tests to iroha-java.

**Pull request merged**